html_css/v2/main-course/custom-properties.md

```diff
@@ -0,0 +1,205 @@
+### Introduction
+
+Custom properties (also known as CSS variables) can be a really useful and powerful tool when writing our CSS files. In short, they allows us to reference a CSS value however many times we want throughout a file. How is this any different or better than just using the value, you ask?
+
+By using custom properties, instead of having to update every single instance of a specific value ("This shade of red is too light, let's change the shade on all seven of these selectors"), we only need to update a single instance: the custom property itself.
```

Comment:
also helps keeping colors consistent throughout a project, which becomes very useful in larger projects.